### PR TITLE
Avoid NPE for summary graph generation

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

File: stress/common/src/main/java/alluxio/stress/common/SummaryStatistics.java [hunk 1]
```diff
@@ -39,6 +41,12 @@
    */
   public SummaryStatistics() {
     // Default constructor required for json deserialization
+    mTimePercentileMs = new float[101];
+    Arrays.fill(mTimePercentileMs, 0);
+    mTime99PercentileMs = new float[StressConstants.TIME_99_COUNT];
+    Arrays.fill(mTime99PercentileMs, 0);
+    mMaxTimeMs = new float[StressConstants.MAX_TIME_COUNT];
+    Arrays.fill(mMaxTimeMs, 0);
```
Comment (bradyoo) [inline stress/common/src/main/java/alluxio/stress/common/SummaryStatistics.java:49]: Can you describe why these are necessary? When are these values missing during json serialization?

**Comment by bradyoo** [comment]
alluxio-bot, check this please

**Comment by bradyoo** [comment]
alluxio-bot, test this please.

File: stress/common/src/main/java/alluxio/stress/common/SummaryStatistics.java (hunk 1 above)
Comment (gpang) [reply stress/common/src/main/java/alluxio/stress/common/SummaryStatistics.java:49]: Sometimes, when there is no data, we just serialize a default constructed instance, and these fields were all `null`.

**Comment by bradyoo** [comment]
LGTM

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**